style/article-2022.css

```diff
@@ -268,6 +270,7 @@ figure p, figure pre { margin-inline-start: 0; margin-inline-end: 0; }
 .inlinedemolink { font-size: 80%; margin-inline-start: 1em; margin-start: 1em;  }
 figcaption { font-style: italic; }
 figcaption:lang(zh) { font-style: normal; }
+figcaption:lang(pes) { font-style: normal; display: inline-block; transform: skew(7deg); padding: 0 2px; -webkit-transform: skew(7deg); -moz-transform: skew(7deg); }
```

Comment:
I see you're using the `transform` property, rather than `font-style oblique`.  That does work with more browsers, so perhaps we should use both until the browsers all support font-style properly? 

I also notice that you're using a 7deg skew here, but 10deg for the cite.  Was that intentional?  If so, i can replicate it in the font-style declarations.

Fwiw, i had used a 14deg slant just because that corresponds to the Latin script default in the Fonts spec.

Reply:
Btw, i haven't uploaded my local version of the .css file (luckily) to GH. If i did, we'd have a conflict, so i'll hang back on that until we decide what to do here.

Currently my local file says things like:

````
cite:lang(pes) {
    font-style: oblique -14deg;
    }
````